routing/routing_exceptions.hpp

```diff
@@ -6,4 +6,5 @@ namespace routing
 {
 DECLARE_EXCEPTION(CorruptedDataException, RootException);
 DECLARE_EXCEPTION(RoutingException, RootException);
+DECLARE_EXCEPTION(MwmIsNotAliveException, RootException);
```

Comment:
How about to create a base class for all routing exceptions? I.e, you may do following:
```c++
DECLARE_EXCEPTION(RoutingException, RootException);
DECLARE_EXCEPTION(MwmIsNotAliveException, RoutingException);
DECLARE_EXCEPTION(CorruptedDataException, RoutingException);
```
?
What do you think?

Reply:
I think it's a good idea. Let's do it in this PR.

Reply:
I overlooked the code. Everything should be ok after this changes.